bin/helpers.rb

```diff
@@ -19,7 +19,5 @@ def web_service_running?
 end
 
 def web_service
-  abort("\n** ABORTED: docker-compose not installed **") unless docker_compose_installed?
-  out, = Open3.capture2("docker-compose ps --services --filter 'status=running' | grep web")
-  out.strip
+  'web'
```

Comment:
??

Reply:
The main version of this script is getting the list of running services using `docker-compose` and re-using the one called "web" (or including "web" in its name). This basically means that in order to run any of our `bin` scripts, you had to manually start all the services with `docker-compose`.

We are removing that dependency. Now we don't check if there is an existing "web" process, we will always be creating new containers from the same image.

Reply:
ok, I guess this method can go away and just use the string `web` in `bin/web` and remove `&& !web_service.empty?` from line 18 in this file